server/src/main/scala/com/cloudera/livy/server/interactive/InteractiveSession.scala

```diff
@@ -292,7 +292,7 @@ class InteractiveSession(
         val path = Seq(sparkHome, "R", "lib", "sparkr.zip").mkString(File.separator)
         val rArchivesFile = new File(path)
         require(rArchivesFile.exists(), "sparkr.zip not found; cannot run sparkr application.")
-        rArchivesFile.getAbsolutePath()
+        rArchivesFile.getAbsolutePath() + "#sparkr"
```

Comment:
Don't you need to add this suffix when the user sets `SPARKR_PACKAGE` too? Avoids having to force the user to do it manually.